### ISSUE TITLE
gtk-3.20 some app title bar not homogen

### ISSUE THREAD

**Issue by S0llarCode** [Jannomag/Yaru-Colors#117]
ManjaroLinux 21.1.2
Gnome 40.4.0
yaru-colors-gtk-theme 21.04
Updated system, but the problem was already present before.

![before](https://i.ibb.co/XJ368NL/Capture-d-cran-du-2021-09-16-12-26-26.png)

The main applications are not affected (firefox, Settings, Files, Tweaks,..).

It is the applications like Telegram, vlc, qbittorrent,... that do not have the same title bar color.

When I change the theme's gtk-3.20 file, it affects the title bars of non-homogeneous applications, but the changes to be made are unknown to me.

**Comment by Jannomag** [comment]
Are those apps installed via Snap or Flatpak?

**Comment by S0llarCode** [comment]
Neither of them.
Apps installed via Pamac, the manjaro's package manager.

**Comment by koplo199** [comment]
Hi, I got the same issue on the Arduino IDE software.
I'm on Ubuntu 21.04, and the inconsistency looks specific to the Yaru-Colors fork, here is how it looks with the official theme `Yaru-dark` :   

![Screenshot from 2021-09-30 21-12-23](https://user-images.githubusercontent.com/85577251/135517232-89271bbf-517a-42a7-8e4c-827ecbf2794f.png)

And how it looks with Yaru-colors (here `Yaru-Purple-Dark`) : 

![Screenshot from 2021-09-30 21-13-05](https://user-images.githubusercontent.com/85577251/135517205-22d1b5b7-51ca-4e79-9cbd-0b0e314cc751.png)

(Arduino IDE downloaded from the [website](https://www.arduino.cc/en/software))

**Comment by Jannomag** [comment]
I noticed the same with Arduino IDE on my laptop and this worked for me:
https://github.com/horst3180/arc-theme/issues/306#issuecomment-162437856

**Comment by koplo199** [comment]
@Jannomag Thanks for the hint, unfortunately that did not work for me, did you also installed Arduino IDE from the website ? I'm wondering what could be the difference between our systems, I also installed `gtk2-engines-murrine` as mentioned in the thread you linked but that also did work with it. Do you have any other idea where it could come from ?

**Comment by Jannomag** [comment]
Did you install a complete color including dark and gtk2?

Try to move the themes either to ~/.local/share/themes or /usr/share/ themes and try again, please

**Comment by koplo199** [comment]
Yes it's solved thank you (I edited my comment in the meantime), your intuition was right moving the theme to `/usr/share/themes` did the trick !

**Comment by Jannomag** [comment]
Great! It’s an issue with Java and it’s the same with every third party gtk theme installed to /home/.
